plover/steno.py

```diff
@@ -21,6 +21,78 @@
 _NUMBERS = set('0123456789')
 _IMPLICIT_NUMBER_RX = re.compile('(^|[1-4])([6-9])')
 
+def compare_normalize_stroke(x, y):
+    for i in range(len(x.strokes)):
+        if i > len(y.strokes) - 1:
+            return 1
+        x_norm = normalize_stroke(x.strokes[i])
+        y_norm = normalize_stroke(y.strokes[i])
+        if less_than(x_norm, y_norm):
+            return -1
+        elif less_than(y_norm, x_norm):
+            return 1
+    if len(x.strokes) < len(y.strokes):
+        return -1
+    return 0
+
+def less_than(x, y):
+    x_right_side = True
+    y_right_side = True
+    for i in range(len(x)):
```

Comment:
Oh there's danger in what I said here. I'll see if I can simplify the problem a little.